code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -292,6 +292,9 @@ GLOBAL_LIST_INIT(strain_list, init_glob_strain_list())
 	gib_chance = 5
 	light_system = MOVABLE_LIGHT
 
+	/// All sources that affect our alpha.
+	var/alpha_sources = list()
+
```

Comment:
Idk about this one, sourced alpha would be a large refactor so I dont wanna be too harsh

Also it´s not using defines for accesses

Some ideas to fix this:
Add sourced alpha at atom level andonly implement your usecases
Just use a multiplier * alpha like most places do